setup.py

```diff
@@ -38,46 +38,43 @@
     },
     python_requires=">=3.6, <3.11",
     install_requires=[
-        # XXX: Temporarily pin asgiref to 3.2.x since testing framework freezes
-        # with version 3.3.x
-        "asgiref~=3.3.1",
+        "asgiref~=3.4.1",
         # Temporarily pin asteval, version >= 0.9.24 is no longer compatible with
         # Python 3.6.
-        "asteval==0.9.23",
-        "async-timeout~=3.0.0",
-        "channels~=3.0.3",
-        "channels_redis~=3.2.0",
+        "asteval==0.9.26",
+        "async-timeout~=4.0.2",
+        "channels~=3.0.4",
+        "channels_redis~=3.3.1",
         # Storage requirement for computing hashes.
         "crcmod",
-        "kubernetes~=12.0.1",
-        "docker~=4.4.4",
+        "kubernetes~=21.7.0",
+        "docker~=5.0.3",
         "Django~=3.2.12",
-        "djangorestframework~=3.12.2",
-        "django-filter~=2.4.0",
-        "django-guardian~=2.3.0",
```

Comment:
It would be nicer to move this line into the last commit.

Reply:
Ah, sure.